src/jit/codegenlinear.h

```diff
@@ -113,6 +113,25 @@ void genPutArgStkSIMD12(GenTree* treeNode);
 #endif // _TARGET_X86_
 #endif // FEATURE_SIMD
 
+#if defined(_TARGET_XARCH_) && !defined(LEGACY_BACKEND)
+void genHWIntrinsic(GenTreeHWIntrinsic* treeNode);
```

Comment:
Just `node` should do, `treeNode` unnecessarily long :)

Reply:
Thanks, will change.